python/src/aiconfig/scripts/aiconfig_cli.py

```diff
@@ -104,6 +114,26 @@ def _run_editor_servers_with_configs(
     return Ok(",".join(server_outcomes.unwrap()))
 
 
+def _start_editor_servers_with_configs(
+    start_config: Result[StartServerConfig, str],
+    cli_config: Result[AIConfigCLIConfig, str],
+) -> Result[str, str]:
+    if not (start_config.is_ok() and cli_config.is_ok()):
+        return Err(
+            f"Something went wrong with configs: {start_config=}, {cli_config=}"
+        )
+
+    server_outcomes = _start_server(
+        start_config.unwrap(), cli_config.unwrap().aiconfigrc_path
+    )
+    if server_outcomes.is_err():
+        return Err(
+            f"Something went wrong with starting the aiconfig server: {server_outcomes=}"
+        )
+
+    return Ok(",".join(server_outcomes.unwrap()))
```

Comment:
nit

`.unwrap()` is like a TODO - let's try to use the chaining API where possible (.map, .and_then, do()...)

This can be done more concisely:

```
return do(
  Ok(",".join(server_outcomes_ok)
  for start_config_ok in start_config
  for cli_config_ok in cli_config
  for server_outcomes_ok in _start_server(
      start_config_ok, cli_config_ok.aiconfigrc_path
  )
)
```